catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js

```diff
@@ -154,50 +154,56 @@ function (ich,Marionette,Backbone,ConfigurationEdit,Service,Utils,wreqr,_,$,moda
             var view = this;
             var service = view.model.get('editConfig');
             if (service) {
-            var fpid = service.get("fpid");
-            var idx = fpid.indexOf("_disabled");
-            if (idx > 0) {
-               service.set("fpid", fpid.substring(0, idx));
-            }
-            service.save().then(function(response) {
-                // check to see if the service corresponds to an existing source
-                // if it does, return the source
-                var existingSource = view.source.get('collection').find(function(item) {
-                    var config = item.get('currentConfiguration');
-                   return (config && config.get('properties').id === service.get('properties').id);
-                });
+                var fpid = service.get("fpid");
+                var idx = fpid.indexOf("_disabled");
+                if (idx > 0) {
+                    service.set("fpid", fpid.substring(0, idx));
+                }
 
-                // Logic to check if the service we are adding is going to be a member of an already existing source;
-                // if it is, we need to disable the new service as soon at is created.
-                if (existingSource && view.mode === 'add' && existingSource.get('currentConfiguration') !== service) {
+                if (_.isUndefined(service.get('properties').id)) {
```

Comment:
This block is the only part that was added in this file.  The rest is reformatting.